src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -266,6 +301,9 @@ public static void LogInformation(string message, params object[] args)
         {
             if (IdentityModelEventSource.Logger.IsEnabled())
```

Comment:
Although, Logger.WriteInformation checks IsEnabled and EventLevel, should we add a condition, so we don't make the call?
```cs
if (IdentityModelEventSource.Logger.IsEnabled() && IdentityModelEventSource.Logger.LogLevel >= EventLevel.Information)
{
}
```

Reply:
Only downside to that (IdentityModelEventSource.Logger.LogLevel >= EventLevel.Informational) condition would be evaluated twice for all messages whose eventLevel >= Informational. Once in LogHelper and again in IdentityModelEventSource.